playground/frontend/lib/pages/embedded_playground/components/embedded_actions.dart

```diff
@@ -58,16 +59,26 @@ class EmbeddedActions extends StatelessWidget {
   void _openStandalonePlayground(PlaygroundController controller) {
     // The empty list forces the parsing of EmptyExampleLoadingDescriptor
     // and prevents the glimpse of the default catalog example.
+
+    final loadingDescriptor = controller.getLoadingDescriptor();
+    final contentDescriptor =
+        loadingDescriptor.whereType<ContentExampleLoadingDescriptor>();
+    final standardDescriptor =
+        loadingDescriptor.whereType<StandardExampleLoadingDescriptor>();
```

Comment:
1. descriptor*s*
2. This does not cover 4 other descriptor types, not to mention possibly future types.

Reply:
fixed